content/docs/command-reference/exp/run.md

```diff
@@ -85,9 +85,10 @@ process finishes or gets interrupted (e.g. with Ctrl + `C`), DVC will
 <abbr>workspace</abbr> (overwriting any further changes done by the stage).
 
 `dvc exp run` again will continue from this point (useful for interrupted runs).
-Use `--reset` to roll-back the workspace to `HEAD` and restart the whole
-experiment. Alternatively, you can use `--rev` to continue from a specific
-(previous) checkpoint.
+Use `--reset` to reset (remove) any existing `checkpoint` outputs in your
+workspace before running the experiment. Note that `--reset` overrides any
+existing `dvc.lock` entries for `checkpoint` outputs. Alternatively, you can use
+`--rev` to continue from a specific (previous) checkpoint.
```

Comment:
Consolidated suggestion for this part:
```suggestion
Use `--reset` to remove `checkpoint` outputs completely in order to restart an
experiment from scratch, regardless of any cached contents (`dvc.lock` is
disregarded) — useful for re-training ML models, for example. Alternatively, you
can use `--rev` to continue from a specific (previous) checkpoint.
```

Reply:
But also the earlier sentence (same section) "Having at least a checkpoint output is needed so that the experiment can later restart based on that output's last cached state." should probably be rephrased.